dev-docs/feature-format-matrix/qmd-files/crossref/float/table/document.qmd

```diff
@@ -32,6 +32,7 @@ _quarto:
           - "a[href=\"#tbl-2\"].quarto-xref"
     dashboard: *dom-tests
     revealjs: # reveal resolves anchors differently, using the section headings instead of the float ids
+      ensureHtmlElements:
```

Comment:
Just a head up that fixing this is breaking the feature format matrix test 
https://github.com/quarto-dev/quarto-cli/actions/runs/9423620780/job/25962387025#step:26:214

So I don't know if revealjs is just not well supported here, or something else but this is probably to fix in another issue. 

This PR just surfaced it as previously there was no test, and it was silently not testing